src/templates/container-hotkey.jsx

```diff
@@ -1,53 +1,53 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { Component, PropTypes } from 'react';
```

Comment:
Also `React.PropTypes` is [deprecated](https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html#migrating-from-react.proptypes) as of React 15.5 where `PropTypes` will be moved in its own separate package:
`import PropTypes from 'prop-types';`